.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+# Describe the bug #
+
+A clear and concise description of what the bug is.
+
+# Steps to reproduce #
+
+Steps to reproduce the behavior:
+
+1. Go to '...'
```

Comment:
This list does not add value from a CSI perspective I feel. There is nothing to click/scroll etc. we should change this to something more appropriate for CSI.

Reply:
Added something for now. But its self explanatory , so shouldnt be needed a documentation as such.